module/VuFind/src/VuFind/Db/Service/AccessTokenService.php

```diff
@@ -131,10 +156,11 @@ public function getNonce(int $userId): ?string
     public function deleteExpired(DateTime $dateLimit, ?int $limit = null): int
     {
         $subQueryBuilder = $this->entityManager->createQueryBuilder();
+        // Delete only tokens with expires set to 1
         $subQueryBuilder->select('CONCAT(a.id, a.type)')
             ->from(AccessTokenEntityInterface::class, 'a')
-            ->where('a.created < :latestCreated')
-            ->setParameter('latestCreated', $dateLimit->format(VUFIND_DATABASE_DATETIME_FORMAT));
+            ->where('a.created < :latestCreated AND a.expires = 1')
```

Comment:
Is it safe to use `= 1` across platforms, or should we use a placeholder and a boolean true to ensure that the framework translates the value consistently everywhere?